From foo@baz Wed Jan 21 08:14:38 PST 2009
Date: Wed, 21 Jan 2009 08:14:38 -0800
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: sxg: remove firmware files from sgx_ethtool.c

From: Greg Kroah-Hartman <gregkh@suse.de>

They are not needed here and only cause build warnings and bloat the
object file.

Cc: Mithlesh Thukral <mithlesh@linsyssoft.com>
Cc: LinSysSoft Sahara Team <saharaproj@linsyssoft.com>
Cc: Christopher Harrer <charrer@alacritech.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/sxg/sxg_ethtool.c |    9 ---------
 1 file changed, 9 deletions(-)

--- a/drivers/staging/sxg/sxg_ethtool.c
+++ b/drivers/staging/sxg/sxg_ethtool.c
@@ -53,15 +53,6 @@
 #include "sxghw.h"
 #include "sxghif.h"
 #include "sxg.h"
-//#include "sxg.c"
-#define SXG_UCODE_DBG			 0  	 /* Turn on for debugging */
-#ifdef SXG_UCODE_DBG
-#include "saharadbgdownload.c"
-#include "saharadbgdownloadB.c"
-#else
-#include "saharadownload.c"
-#include "saharadownloadB.c"
-#endif
 
 struct sxg_nic_stats {
         char stat_string[ETH_GSTRING_LEN];
